docs/release-notes/self-hosted/self-hosted-v6.0.0.md

```diff
@@ -0,0 +1,131 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v6.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/5.2.1
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/5.2.1
+---
+
+# Self-hosted v6.0.0
+
+These release notes are for [Codacy Self-hosted v6.0.0](https://github.com/codacy/chart/releases/tag/6.0.0){: target="_blank"}, released on March 1, 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v6.0.0:
+
+1.  [Check the release notes](../index.md#self-hosted) for all Codacy Self-hosted versions between your current version and the most recent version for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](../../chart/maintenance/upgrade.md).
+
+1.  Update your Codacy command-line tools to the versions with the Git tag `self-hosted-6.0.0`:
+
+    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-6.0.0)
+    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-6.0.0)
+
+## Breaking changes
+
+This version of Codacy Self-hosted introduces the following breaking changes:
+
+-   [Dropped support for Kubernetes 1.14 and MicroK8s 1.14](https://docs.codacy.com/v6.0/chart/requirements/)
+
+    Kubernetes 1.14 is an old version [first released in 2019](https://kubernetes.io/blog/2019/03/25/kubernetes-1-14-release-announcement/){: target="_blank"} that's no longer being maintained.
+
+-   [Removed the tool **PMD (Legacy)**](../cloud/cloud-2022-02-16-pmd-legacy-removal.md)
+
+    If you're currently using this tool to analyze your repositories and want to start using **PMD** instead, we recommend that you follow the next steps **before upgrading to this Codacy Self-hosted version**:
+
+    1.  Take note of the code patterns that you have enabled on **PMD (Legacy)** and enable the corresponding code patterns on the new tool **PMD**.
+
+        The new version of the tool includes different code patterns, so it's possible that not all code patterns will have a direct match on the new tool.
+
+    1.  Disable **PMD (Legacy)** and enable **PMD**.
+
+-   Dropped support for custom rules in PHP Mess Detector configuration files
+
+    Now, any custom rules are filtered out from PHP Mess Detector configuration files before running the tool. This was done to protect Codacy's infrastructure against potential arbitrary code <span class="skip-vale">execution</span> vulnerabilities.
```

Comment:
This can have an impact on customers that were using configuration files with custom rules. However, I'm not sure if we should include this or not.

Reply:
Risk of having customers complaining about a missing feature vs risk of having someone try to target the other tools with the same vulnerability?

Reply:
Exactly, @lventura-codacy. :+1:

Reply:
After confirming with Lorenzo we have indeed only verified and fixed a couple of tools, leaving this in the changenotes may open a small pandora's box

Reply:
I'm removing this now, thanks for the help!